core/domain/exp_jobs_one_off_test.py

```diff
@@ -565,6 +609,272 @@ def test_nonhuman_committers_not_counted(self):
+
+        self.set_admins([self.ALBERT_NAME])
+        owner = user_services.UserActionsInfo(self.albert_id)
+
+        rights_manager.publish_exploration(owner, self.VALID_EXP_ID)
+
+        # Start ExplorationValidityJobManager job on published exploration.
+        job_id = exp_jobs_one_off.ExplorationValidityJobManager.create_new()
+        exp_jobs_one_off.ExplorationValidityJobManager.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        actual_output = exp_jobs_one_off.ExplorationValidityJobManager.get_output(job_id) # pylint: disable=line-too-long
+
+        self.assertEqual(actual_output, [])
+
+    def test_strict_validation_errors_are_raised_for_published_exploration(
+            self):
+        """Checks validation errors are not present for valid exploration."""
+        exploration = exp_domain.Exploration.create_default_exploration(
+            self.VALID_EXP_ID, title='title', category='category')
+
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        # Start ExplorationValidityJobManager job on unpublished exploration.
+        job_id = exp_jobs_one_off.ExplorationValidityJobManager.create_new()
+        exp_jobs_one_off.ExplorationValidityJobManager.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        actual_output = exp_jobs_one_off.ExplorationValidityJobManager.get_output(job_id) # pylint: disable=line-too-long
+        self.assertEqual(actual_output, [])
+
+        self.set_admins([self.ALBERT_NAME])
+        owner = user_services.UserActionsInfo(self.albert_id)
+
+        rights_manager.publish_exploration(owner, self.VALID_EXP_ID)
+
+        # Start ExplorationValidityJobManager job on published exploration.
+        job_id = exp_jobs_one_off.ExplorationValidityJobManager.create_new()
+        exp_jobs_one_off.ExplorationValidityJobManager.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        actual_output = exp_jobs_one_off.ExplorationValidityJobManager.get_output(job_id) # pylint: disable=line-too-long
+        expected_output = [(
+            '[u\'exp_id0\', '
+            '[u\'This state does not have any interaction specified.\']]'
+        )]
+        self.assertEqual(actual_output, expected_output)
+
+        exploration.states['Introduction'].update_interaction_id(
+            'TextInput')
+
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        rights_manager.publish_exploration(owner, self.VALID_EXP_ID)
+
+        # Start ExplorationValidityJobManager job on published exploration.
+        job_id = exp_jobs_one_off.ExplorationValidityJobManager.create_new()
+        exp_jobs_one_off.ExplorationValidityJobManager.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        actual_output = exp_jobs_one_off.ExplorationValidityJobManager.get_output(job_id) # pylint: disable=line-too-long
+        expected_output = [(
+            '[u\'exp_id0\', '
```

Comment:
The u in front is from the python prints? I think there is a way (not sure) to get the raw string output from the job. Could you look at other one_off_test files to see how output from a job is handled? If there isn't a way to do it, lets leave it as is.

Reply:
Ah I see. I am fine to leave it as is... Thanks for confirming!